src/apps/games/tests/bad_libmagic_npz.py

```diff
@@ -0,0 +1,174 @@
+# This particular npz file is valid but libmagic detects it as a DOS boot sector instead of a zip file
+base64data=b"""UEsDBBQAAgAIABcHhFHviuZpaAcAAMC+AAAVAAAAYmluYXJ5SW5wdXROQ0hXUGFja2Vk7d3Pb9xE
```

Comment:
Weird, why that ?

https://github.com/threatstack/libmagic/blob/1249b5cd02c3b6fb9b917d16c76bc76c862932b6/magic/Magdir/filesystems#L542 ?

Reply:
Because libmagic rules are conflicting and sometimes buggy. If you search online you can find other reports of different cases for different kinds of files where libmagic gets it wrong.

It seems very very hard to always get it right because if you have 1000s of different kinds of files, and each of them has different magic byte rules, well, sometimes those rules will be ambiguous and come into conflict.

Reply:
So that's why we specialize for the specific 3 file types we care about. Since we only deal with 3, we can be confident that our custom magic byte checks will never have false negatives or conflict.

Reply:
Then maybe we can remove libmagic alltogether ?

Reply:
And simplify docker image to remove libmagic

Reply:
It's still useful for generating error messages when an invalid file or wrong format happens - it allows the error message to say the type of the file when it doesn't match, and is still more than 99.9% perfect accurate. 99.9% accurate is not good enough for validation, but it is good enough for error messages, so I think it's okay to keep.